### ISSUE TITLE
Announcement: journal EOL

### ISSUE THREAD

**Issue by lukebarnard1** [lukebarnard1/journal#118]
**TLDR**: Regrettably, I don't have enough time to implement journal in it's entirety and there are decent alternative solutions. 

-----

journal is exactly where I left it a few months ago, looking a bit dated and not being easy to use and not having the features I wanted in my blog site.

Meanwhile, the main purpose of journal remains unfulfilled. The work to expose a matrix room as a journal blog was never done as I haven’t had the time or the will to go and make it happen.

My original decision to replace the client-based version of journal was a little short-sighted. It’s true that it took a long time to load a blog, but it worked and had live updates and allowed posting of blogs and comments. In hindsight I could have enhanced this with a server that exposed a room as a set of markdown files to be consumed by the frontend. 

But when you think about it, trying to optimise the bit that content management systems have already solved would also have been a bit short-sighted.

The other way of looking at it is that if the client-based journal could have avoided doing the main blog article bit, it might have actually been a useable solution and a widely compatible one. For example, it could have been embedded into a blog that was already deployed via a CMS. 

Content management is a well developed area of open source, with a lot of different options (e.g. Ghost, Hugo, Jekyll), all that produce static content in a customisable way. Any solution for blogs on matrix should clearly make use of the work that has been done here. 

Disqus is an example of a centralised discussion thread embed which I think is an obvious solution for the matrix community to clone. The matrix equivalent could be highly customisable and integrate with popular matrix clients via a “matrix.to” link (e.g. “Open discussion in Riot.im”).

For other features that I intended to implement with journal, there are existing solutions that are already widely used. For example, twitter, mastodon or email can be used to notify readers of a new blog (or a message in a matrix room, which could even be the start of the embedded discussion thread!).

Regardless of the direction I took journal in, I clearly didn’t have time to maintain such a project in it’s entirety. My current plan is to stop maintaining it, eventually replace my personal blog with Hugo and who knows, maybe I’ll have a go at making a Disqus clone for Matrix one day.

Thanks for reading and showing interest in this project. 

Happy decentralising 😄
